gapis/api/vulkan/vulkan.api

```diff
@@ -2929,9 +2929,9 @@ cmd VkResult vkAllocateMemory(
     MappedOffset:     0,
     MappedSize:       0,
     MappedLocation:   null,
-    MemoryTypeIndex:  allocateInfo.memoryTypeIndex,
-    Data:             make!u8(allocateInfo.allocationSize),
+    MemoryTypeIndex:  allocateInfo.memoryTypeIndex
   )
+  memoryObject.Data = make!u8(allocateInfo.allocationSize)
```

Comment:
I assume you've changed this to avoid another template change?

Reply:
Yeah, I was having a hard time trying to figure out where exactly was the right place to make this change in the templates. It seemed easier to just avoid it, since avoiding the assignment is nice and easy.